Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -92,6 +92,34 @@ extension Issue {
     let issue = Issue(kind: .unconditional, severity: severity, comments: Array(comment), sourceContext: sourceContext)
     return issue.record()
   }
+  
+  /// Record an issue when a running test fails unexpectedly.
+  ///
+  /// - Parameters:
+  ///   - comment: A comment describing the expectation.
+  ///   - severity: The severity of the issue.
+  ///   - sourceLocation: The source location to which the issue should be
+  ///     attributed.
+  ///   - kind: The kind of the issue.
+  ///
+  /// - Returns: The issue that was recorded.
+  ///
+  /// Use this function if, while running a test, an issue occurs that cannot be
+  /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
+  /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
+  @_spi(Experimental)
+  @_spi(ForToolsIntegrationOnly)
+  @discardableResult public static func record(
+    kind: Kind = .unconditional,
```

Comment:
Ok now that this is a distinct overload, I think you'll need to make `kind:` _not_ have a default value at all, because all parameters have a default value now and `Issue.record()` (zero args) may be ambiguous with the public API
```suggestion
    kind: Kind,
```